docs/src/documentation/03-components/05-input/select/03-accessibility.mdx

```diff
@@ -83,4 +99,54 @@ When using the `help` or `error` props, their content is set as `aria-describedb
 />
 ```
 
-It would have the screen reader announce: “Nationality. Required field.”.
+It would have the screen reader announce: "Nationality. Required field."
+
+If you provide both an `ariaDescribedby` prop and use `error` or `help`, the component automatically combines them, ensuring all descriptive content is properly announced:
+
+```jsx
+<div>
+  <p id="terms-info">Please review our terms and conditions.</p>
+  <Select
+    options={termsOptions}
+    value={termsValue}
+    onChange={onChange}
+    label="Accept Terms"
+    error="This field is required"
+    ariaDescribedby="terms-info"
+  />
+</div>
+```
+
+In this example, both the "terms-info" text and the error message will be announced by screen readers.
+
+### InputGroup Integration
+
+When using Select within an InputGroup, the `aria-describedby` association follows these rules:
+
+#### Example 1
+
+If the InputGroup has `error`/`help` messages, these will be properly associated with all child components:
+
+```jsx
+<InputGroup label="Travel preferences" error="Please complete all fields">
+  <Select options={countryOptions} label="Departure country" />
+  <Select options={countryOptions} label="Destination country" />
+</InputGroup>
+```
+
+In this example, all Select components will have the InputGroup's error message "Please complete all fields" announced by screen readers.
+
+#### Example 2
+
+If individual Select components have their own `error`/`help` messages (and the InputGroup doesn't), only those specific components will have `aria-describedby` set:
+
+```jsx
+<InputGroup label="Travel preferences">
+  <Select options={countryOptions} label="Departure country" />
+  <Select options={countryOptions} label="Destination country" error="This field is required" />
+</InputGroup>
+```
+
+In this example, only the second Select will have its error message "This field is required" announced.
+
+Avoid setting `ariaDescribedby` directly on Select components when inside an InputGroup, as these values will be overwritten by the InputGroup's internal accessibility logic.
```

Comment:
I think that's quite important.

Would it make sense to give Example 3 with this situation and explain what will be announced by the screen reader?

Reply:
Good point. I added third example, I hope it's enough like this.

I also did changes to previous examples, I wanted to replace the native elements with our components but instead I kept the elements and hide them to be consistent with our other examples, e.g. here in Seat https://orbit.kiwi/components/visuals/seat/accessibility/